src/frontend/pages/ChangeRequestsPage/change-requests-filter/change-requests-filter.tsx

```diff
@@ -6,7 +6,7 @@
 import { useState } from 'react';
 import { Button, Card, Dropdown, Form } from 'react-bootstrap';
 import { ChangeRequestType, ChangeRequestReason } from 'utils';
-import styles from './change-requests-filter.module.css';
+import styles from './_change-requests-filter.module.scss';
```

Comment:
why the underscore?

Reply:
I did it because it's general Saas naming ocnvention but turns out it's only for "partial" files which is not really what we're using so I'll get rid of it